File: CurrencyConverter/app/src/main/java/com/example/practiceapp/MainActivity.java
package com.example.practiceapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    private EditText currency_in_dollar;
    private Button convert;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        currency_in_dollar = findViewById(R.id.currency_to_converter);
        convert = findViewById(R.id.convert_btn);

        convert.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Double currency = Double.valueOf(currency_in_dollar.getText().toString());
                Toast.makeText(MainActivity.this,""+currency * 75.61,Toast.LENGTH_SHORT).show();
            }
        });

    }
}

File: CurrencyConverter/app/src/main/java/com/example/googlesummerofcode/MainActivity.kt
package com.example.googlesummerofcode

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Handler().postDelayed({
            val homeIntent = Intent(this@MainActivity, LoginActivity::class.java)
            startActivity(homeIntent)
            finish()
        }, MainActivity.Companion.SPLASH_TIME_OUT.toLong())
    }

    companion object {
        private const val SPLASH_TIME_OUT = 4000
    }
}